migrate/db-seed-commonjs-pkg/run.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+set -eux
+
+yarn install
+
+# generate not needed, it will be done by db push
+# yarn prisma generate
+yarn prisma db push --force-reset
```

Comment:
if `force-reset` does not work for whatever reason, the test would possibly also succeed if the seeding fails, correct?

Reply:
Well here it's about preparing the db and I assume test.sh runs after with
```
#!/bin/sh

set -eux

yarn prisma db pull --print

yarn prisma db seed

yarn test
```

This should fail